Doc/library/os.path.rst

```diff
@@ -324,9 +324,9 @@ the :mod:`glob` module.)
 
 .. function:: normcase(path)
 
-   Normalize the case of a pathname.  On Unix and Mac OS X, this returns the
-   path unchanged; on case-insensitive filesystems, it converts the path to
-   lowercase.  On Windows, it also converts forward slashes to backward slashes.
+   Normalize the case of a pathname.  On Windows, converts all characters in the
+   pathname to lowercase, and also converts forward slashes to backward slashes.
+   On other operating systems, return the path unchanged.
```

Comment:
IMO the middle sentence either needs a subject (_this_ converts . . .), like the old sentence had, or it needs changing to match the imperative (?) mood of the first and last sentences (_convert_ all characters . . .).

Reply:
+1 to imperative

Reply:
Right, I am going to change "converts" to "convert".